src/Features/ERDDAP/Platform/Observations/CurrentConditions/wind.tsx

```diff
@@ -72,12 +72,20 @@ interface DisplayWindCardProps extends WindCardProps {
   datasets: DataTimeSeries[]
   unitSystem: UnitSystem
   timeSeries: PlatformTimeSeries[]
+  startTime?: Date
+  endTime?: Date
 }
 
 /**
  * Display wind datasets on a current conditions card
  */
-export const DisplayWindCard: React.FunctionComponent<DisplayWindCardProps> = ({ platform, datasets, unitSystem }) => {
+export const DisplayWindCard: React.FunctionComponent<DisplayWindCardProps> = ({
```

Comment:
Codacy found an issue: [Arrow function has a complexity of 8. Maximum allowed is 4.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6263729)